### ISSUE TITLE
How about add a web UI for colly?

### ISSUE THREAD

**Issue by hantmac** [gocolly/colly#297]
- Just like pyspider, how about add a web UI for Colly which can help user to control their spider.

**Comment by asciimoo** [comment]
This is a great idea. I started to implement a web debugger: https://github.com/gocolly/colly/blob/389bc3c811e16f7bfba497d8dc6bf4f98a51c8fe/debug/webdebugger.go , but it's really basic and I'm still not sure if it is the right approach. Would you like to work on this topic?

**Comment by hantmac** [comment]
- First of all, we should use the separation of front and back ends; 
- secondly, jQuery is not a good way to build the front end, recommend [Vue.js](https://vuejs.org/v2/guide/index.html);
- then,we call API which provided by Colly  in front end.

How about it?

By the way, where could I find the API document of Colly?

**Comment by asciimoo** [comment]
> First of all, we should use the separation of front and back ends;

I agree, that's why I created a debug interface which can be used by the webdebugger.

> secondly, jQuery is not a good way to build the front end, recommend Vue.js;

Also agreed, it was just a quick POC.

> By the way, where could I find the API document of Colly?

https://godoc.org/github.com/gocolly/colly

**Comment by hantmac** [comment]
Hi, I made a simple web ui long time ago (just omit the picture ), now I wanna to make it open source.
And I think it is a good fit for Colly's management page.
- `client` is the VM which one colly deploy on.
- `project` is all your colly project.
We only need to use your colly lib to make web api. What do you think? Listen to your opinions with humility.
<img width="1661" alt="2019-03-04 11 06 42" src="https://user-images.githubusercontent.com/7600925/53708482-ab227e80-3e6e-11e9-9877-f07798533810.png">

If use wanna review the source code, I think we can put it into colly project.

**Comment by vosmith** [comment]
UI looks nice and clean :+1: is there a place we can play with a demo?

**Comment by hantmac** [comment]
> UI looks nice and clean 👍 is there a place we can play with a demo?

wait a minute, I am making it and the I will push it to github.

**Comment by hantmac** [comment]
@vosmith The project is https://github.com/hantmac/CollyFront, I named it `CollyFront`, if you mind it, I can rename it.☺️😆

**Comment by vosmith** [comment]
@hantmac awesome I'll check it out there!

**Comment by tycoons** [comment]
great idea you will have a little doc or demo even brider to see the beast in action :) because I have a doubt about the implementation with go colly
thank you for all his
